src/backend/marsha/core/serializers/playlist.py

```diff
@@ -26,15 +30,41 @@ class Meta:
             "users",
         ]
 
-    def create(self, validated_data):
-        """Create the Playlist object with the passed data."""
-        return Playlist.objects.create(**validated_data)
+    portable_to = serializers.SerializerMethodField(read_only=False)
+
+    def get_portable_to(self, obj):
+        """Getter for portable_to attribute instead of PlaylistSerializer to prevent recursion."""
+        return obj.portable_to.values("id", "title")
+
+    def to_internal_value(self, data):
+        """
+        Validate portable_to.
+
+        Raise a ValidationError if portable_to contains current object playlist id
+        or an unknown playlist id.
+        """
+        value: OrderedDict = super().to_internal_value(data)
+        if "portable_to" in data.keys():
+            reachable_from_playlist_ids = data.get("portable_to")
+            for playlist_id in reachable_from_playlist_ids:
+                try:
+                    Playlist.objects.get(pk=playlist_id)
+                except Playlist.DoesNotExist as error:
+                    raise serializers.ValidationError(
+                        {"playlist_not_exists": _("Playlist not exists.")}
```

Comment:
`playlist does not exists` ? `playlist not existing` ? Maybe specifying the non existing playlist ?